java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -73,4 +73,21 @@ public interface BitmapBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> bitcount(String key, long start, long end, BitmapIndexType options);
+
+    /**
+     * Returns the bit value at <code>offset</code> in the string value stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/getbit/">redis.io</a> for details.
+     * @param key The key for the string.
+     * @param offset The index of the bit to return.
+     * @return The bit at offset of the string. Returns zero if the key is empty or if the positive
+     *     <code>offset</code> exceeds the length of the string. An error is thrown when the <code>
+     *     offset</code> is negative.
+     * @example
+     *     <pre>{@code
+     * Long payload = client.getbit("myKey1", 1).get();
```

Comment:
we can expend the example a bit
like setting a value and getting a specific bit from the value
if we set 'A' which is 01100001 in binary, we can have a better explanation of why this returns 1 :)